Scripts/application.py

```diff
@@ -0,0 +1,58 @@
+import sqlite3
+import json
+from typing import Dict
+from flask import Flask
+app = Flask(__name__)
+
+@app.route('/')
+def hello_world():
+    return 'Hello World!'
+    
+@app.route('/api/v1/person')
+def return_people():
+    # returns a json of all people
+    with sqlite3.connect('../db/allYourSantaAreBelongToUs.db') as connection:

```

Comment:
I'm seeing '../db/allYourSantaAreBelongToUs.db' in multiple places. Can you store it as a constant?

Reply:
yes